modules/ITK/cli/statismo-build-gp-model.cxx

```diff
@@ -136,7 +136,16 @@ bool isOptionsConflictPresent(programOptions& opt) {
         return true;
     }
 
-    if (opt.strOutputFileName == "" || opt.strReferenceFile == "") {
+    if (opt.strOptionalModelPath == "" && opt.strReferenceFile == "") {
+        return true;
+    }
+
+    if (opt.strOptionalModelPath != "" & opt.strReferenceFile != "") {
+        return true;
+    }
```

Comment:
True, this should be written din one statement to make it more clear. 
I prefer, however, to simply or the two statements, rather than using the (bitwise) xor.